pybots/web/static/github.css

```diff
@@ -0,0 +1,386 @@
+body {
+    font-family: Helvetica, arial, sans-serif;
+    font-size: 14px;
+    line-height: 1.6;
+    background-color: white;
+    padding: 30px;
+    color: #333;
+}
+
+body > *:first-child {
+    margin-top: 0 !important;
+}
+
+body > *:last-child {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The universal selector (*) is known to be slow.](https://www.codacy.com/app/thejoeejoee/pybots-server/file/2266725696/issues/source?bid=3184258&fileBranchId=3184263#l14)